install/upgrade/update_to_dev.php

```diff
@@ -180,7 +180,30 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addKey($table, 'plugin_flyvemdm_policycategories_id', 'plugin_flyvemdm_policycategories_id');
    // All policies exist for Android
    $migration->addPostQuery("UPDATE `$table` SET `is_android_policy` = '1'");
-   $migration->addPostQuery("DELETE FROM `$table` WHERE `symbol` IN ('disableFmRadio','disableVoiceMail','disableCallAutoAnswer','disableVoiceDictation','disableUsbOnTheGo')");
+   $policies = [
+      'disableFmRadio',
+      'disableVoiceMail',
+      'disableCallAutoAnswer',
+      'disableVoiceDictation',
+      'disableUsbOnTheGo',
+   ];
+   $tasksTable = 'glpi_plugin_flyvemdm_tasks';
+   $request = [
+      'FIELDS'     => [$table => ['symbol'], $tasksTable => ['plugin_flyvemdm_fleets_id']],
+      'FROM'       => $table,
+      'INNER JOIN' => [
+         $tasksTable => ['FKEY' => [$tasksTable => 'plugin_flyvemdm_policies_id', $table => 'id']],
+      ],
+      'WHERE'      => ['symbol' => $policies],
+   ];
+   $result = $DB->request($request);
+   if (count($result) > 0) {
+      $mqttClient = PluginFlyvemdmMqttclient::getInstance();
+      foreach ($DB->request($result) as $row => $data) {
+         $mqttClient->publish($data['plugin_flyvemdm_fleets_id'] . "/Policy/" . $data['symbol'], null, 0, 1);
```

Comment:
There are some changes in the MQTT topics, and some others pending (leading slash)

Currently a topic has this structure

```/0/fleet/38/Policy/storageEncryption/Task/37```
- entity ID
- litteral **fleet**
- fleet ID
- litteral **Policy**
- symbol of a policy
- litteral **Task**
- task ID

Moreover, the leading slash in the topic is not yet removed (PR #315 issue #313).

You may pick some more columns in your request to g et the missing IDs and complete the MQTT topic.